site/fetchers/util/events.js

```diff
@@ -17,31 +18,24 @@ export function splitEvents({
     const endDateTime = parseDateAndResetTime(event.endDateTime.iso);
 
     // In a rare case of user error we omit this event from the output list
-    if (
-      (!dateFns.isSameDay(startDateTime, endDateTime) &&
-        !dateFns.isBefore(startDateTime, endDateTime)) ||
-      dateFns.differenceInMinutes(endDateTime, startDateTime) < 0
-    ) {
+    if (startDateTime.isAfter(endDateTime, 'minute')) {
       return false;
     }
 
     switch (timeline) {
       case 'today':
-        if (dateFns.isSameDay(startDateTime, endDateTime)) {
-          return dateFns.isSameDay(startDateTime, todayDateTime);
+        if (startDateTime.isSame(endDateTime, 'day')) {
+          return startDateTime.isSame(todayDateTime, 'day');
         }
-        return dateFns.isWithinRange(todayDateTime, startDateTime, endDateTime);
+        return moment(todayDateTime).isBetween(startDateTime, endDateTime);
       case 'past':
         return (
-          !dateFns.isWithinRange(todayDateTime, startDateTime, endDateTime) &&
-          dateFns.isBefore(endDateTime, todayDateTime) &&
-          !dateFns.isSameDay(endDateTime, todayDateTime)
+          !moment(todayDateTime).isBetween(startDateTime, endDateTime) &&
```

Comment:
Out of interest what was the reason for switching to moment? Is it smaller these days?

I've found date-fns nice, but lacking in some functionality on a previous project. So I've got nothing against moment.

Reply:
We were using both. I have no experience with any of them. It looked easier to refactor this way. But I basically just flipped a coin. 😄